{{cookiecutter.project_slug}}/_/frameworks/Django/requirements.txt

```
@@ -22,5 +22,5 @@ pytz==2020.1              # via django
 sentry-sdk==0.16.0        # via -r requirements.in{% endif %}
 sqlparse==0.3.1           # via django
 {%- if cookiecutter.monitoring == 'Sentry' %}
-urllib3==1.25.9           # via sentry-sdk{% endif %}
+urllib3==1.25.10           # via sentry-sdk{% endif %}
```

The change should keep the indenting intact.
```suggestion
urllib3==1.25.10          # via sentry-sdk{% endif %}
```